### ISSUE TITLE
Zero accessibility issues causes test to fail

### ISSUE THREAD

**Issue by ediblecode** [nhsevidence/wdio-cucumber-steps#13]
### Repro steps

Find a page with no accessibility issues and write a feature using the accessibility step definition using Axe, e.g.:

```
Scenario: Accessibility checking
    Given I open the url "/some-url"
    Then the page should have no accessibility issues
```

### Expected result

The test passes

### Actual result

The test fails with "AssertionError: Found 0 accessibility errors on http://host:port/some-url: expected [] to equal []"
